src/scripts/createBlock.ts

```diff
@@ -13,30 +13,41 @@ import { getBlockchain } from "../blockchain";
 import {
   operationsCollectionFilters as filters,
   get,
-  operationsCollection
+  operationsCollection,
+  getVideoMultiHashForUid
 } from "../firestore";
 import {
   BLOCKCHAIN_VERSION_NO,
+  VirtualBlock,
+  StellarMetadata
+} from "../schema";
+import {
   Block,
-  StellarMetadata,
-  IpfsBlock,
   Operation,
-  RequestInvite,
-  Trust
-} from "../schema";
+  RequestInviteData,
+  TrustData,
+  Uid
+} from "../schema/blockchain/version_01";
+
+const BLOCK_FORMAT_SPACES = 2;
 
 /**
  * A map of functions that build the data component of Blockchain Operations from
  * the data component of Firestore Operations.
  */
 const blockchainOpDataBuilders = {
```

Comment:
This can be made more typesafe by using the operation type enum and a mapping type from the enum to the resulting data, like I did in the API definition in the web app, but for now it's fine as is